project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
 
 
   private val workbenchLibsHash = "d619ea2a-SNAP"
-  val serviceTestV = s"0.20-$workbenchLibsHash"
```

Comment:
@Qi77Qi I didn't want to risk messing with anything you were trying to do by using the `SNAP` version so I directly pointed to the particular `serviceTest` version.

Reply:
the reason why this was `SNAP` was becuz I didn't want to wait for https://github.com/broadinstitute/workbench-libs/pull/797 to merge Mon night before I merge leo fix (otherwise auto -tests would've failed all night Mon)....

in other words, we should use latest wb-libs hash

Reply:
Gotcha 👍🏼  Updated accordingly.